OWNERS

```diff
@@ -6,6 +6,7 @@ reviewers:
   - jianzhangbjz
   - tomasdavidorg
   - LuboTerifaj
+  - cardelea
```

Comment:
Is this correct? Shouldn't it be `cardelearht`?

Reply:
@LuboTerifaj   it shod be cardelearht my bad